### PR TITLE
[DDB] Official D&D Beyond Content

### PR DESCRIPTION
**Pull request by FreshRoyal**
This PR includes what little player options are in DDB-exclusive content. Sources included are the three Monstrous Compendium volumes and the Legendary Magic Items released to promote Honor Among Thieves. I also included the mundane boomerang item. I have not added the cover art or the meta info for the XML files, though.

Edit: Cover art and meta info have been added.

### CODE REVIEW

File: supplements/dnd-beyond/monstrous-compendium-2.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			<set name="keywords">mace, evil</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="mace">Weapon</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Legendary</set>
+			<set name="enhancement">3</set>
+			<set name="weapon">ID_INTERNAL_WEAPON_GROUP_MACES</set>
+		</setters>
+		<rules>
```
Comment (AngryGuineaPig) [inline supplements/dnd-beyond/monstrous-compendium-2.xml:42]: The second part of Blessing of the Dragon Queen where the player can choose a damage type to gain immunity to has not been implemented

Comment (FreshRoyal) [reply supplements/dnd-beyond/monstrous-compendium-2.xml:42]: Should those be implemented as a class feature selection?

**Review by AngryGuineaPig — APPROVED** [review]
I believe with the changes i requested completed that everything is all good in these files

**Review by MeriadarGoblin — APPROVED** [review]

File: supplements/dnd-beyond/monstrous-compendium-2.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+		</rules>
+	</element>
+	<element name="Acid" type="Class Feature" source="Internal" id="ID_WOTC_MCV2_MAGIC_ITEM_NIGHTBRINGER_ACID">
+		<compendium display="false" />
+		<description>
+			<p>You gain immunity to acid damage.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-2.xml:57]:
```suggestion
		<sheet display="false"/>
```

File: supplements/dnd-beyond/monstrous-compendium-2.xml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		</rules>
+	</element>
+	<element name="Cold" type="Class Feature" source="Internal" id="ID_WOTC_MCV2_MAGIC_ITEM_NIGHTBRINGER_COLD">
+		<compendium display="false" />
+		<description>
+			<p>You gain immunity to cold damage.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-2.xml:69]:
```suggestion
		<sheet display="false"/>
```

File: supplements/dnd-beyond/monstrous-compendium-2.xml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+		</rules>
+	</element>
+	<element name="Fire" type="Class Feature" source="Internal" id="ID_WOTC_MCV2_MAGIC_ITEM_NIGHTBRINGER_FIRE">
+		<compendium display="false" />
+		<description>
+			<p>You gain immunity to fire damage.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-2.xml:81]:
```suggestion
		<sheet display="false"/>
```

File: supplements/dnd-beyond/monstrous-compendium-2.xml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+		</rules>
+	</element>
+	<element name="Lightning" type="Class Feature" source="Internal" id="ID_WOTC_MCV2_MAGIC_ITEM_NIGHTBRINGER_LIGHTNING">
+		<compendium display="false" />
+		<description>
+			<p>You gain immunity to lightning damage.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-2.xml:93]:
```suggestion
		<sheet display="false"/>
```

File: supplements/dnd-beyond/monstrous-compendium-2.xml [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+		</rules>
+	</element>
+	<element name="Poison" type="Class Feature" source="Internal" id="ID_WOTC_MCV2_MAGIC_ITEM_NIGHTBRINGER_POISON">
+		<compendium display="false" />
+		<description>
+			<p>You gain immunity to poison damage.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-2.xml:105]:
```suggestion
		<sheet display="false"/>
```

File: supplements/dnd-beyond/monstrous-compendium-3.xml [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+			<set name="script"></set>
+		</setters>
+	</element>
+
+	<element name="Blaze Rod" type="Magic Item" source="Monstrous Compendium Volume Three: Minecraft Creatures" id="ID_WOTC_MCV3_MAGIC_ITEM_BLAZE_ROD">
+		<description>
+			<p>The rod sheds dim light in a 5-foot radius. As an action, a creature can try to snap the rod, doing so with a successful DC 14 Strength check. The snapped rod releases its fiery energy in a 5-foot-radius sphere centered on itself. Each creature in that area must make a DC 14 Dexterity saving throw, taking 6 (1d12) fire damage on a failed save, or half as much damage on a successful one.</p>
+		</description>
+		<setters>
+			<set name="keywords">fire, damage</set>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-3.xml:41]:
```suggestion
			<set name="keywords">action, strength, check, fire, damage, dexterity, save, saving throw</set>
```

File: supplements/dnd-beyond/monstrous-compendium-3.xml [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+			<set name="rarity">Unknown</set>
+		</setters>
+	</element>
+
+	<element name="End Crystal" type="Magic Item" source="Monstrous Compendium Volume Three: Minecraft Creatures" id="ID_WOTC_MCV3_MAGIC_ITEM_END_CRYSTAL">
+		<description>
+			<p>An <i>End crystal</i> is a Large object that has Armor Class 11; 30 hit points; vulnerability to bludgeoning and thunder damage; resistance to piercing and slashing damage; and immunity to poison and psychic damage. When a crystal is destroyed, it explodes in a 20-foot-radius sphere of destructive energy. Any creature in that area must make a DC 15 Dexterity saving throw, taking 21 (6d6) force damage on a failed save, or half as much damage on a successful one.</p>
+		</description>
+		<setters>
+			<set name="keywords">force, damage</set>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-3.xml:55]:
```suggestion
			<set name="keywords">dexterity, save, saving throw, force, damage</set>
```

File: supplements/dnd-beyond/monstrous-compendium-3.xml [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+			<set name="traits">ID_WOTC_MCV3_COMPANION_TRAIT_WOLF_OF_THE_OVERWORLD_PACK_TACTICS</set>
+			<set name="actions">ID_WOTC_MCV3_COMPANION_ACTION_WOLF_OF_THE_OVERWORLD_BITE</set>
+		</setters>
+		<rules>
+			<stat name="companion:ac" value="13" />
+			<stat name="companion:hp:max" value="11" bonus="base" />
+			<stat name="companion:initiative" value="2" />
+			<stat name="companion:speed" value="40" bonus="base" />
+			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="base" />
+			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="base" />
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-3.xml:105]:
```suggestion
			<stat name="companion:speed" value="40" bonus="base" />
			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="base" />
			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="double" />
```

File: supplements/dnd-beyond/monstrous-compendium-3.xml [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+		<description>
+			<p>The wolf has advantage on attack rolls against a creature if at least one of the wolf’s allies is within 5 feet of the creature and the ally doesn’t have the incapacitated condition.</p>
+		</description>
+		<sheet>
+			<description>The wolf has advantage on attack rolls against a creature if at least one of the wolf’s allies is within 5 feet of the creature and the ally doesn’t have the incapacitated condition.</description>
+		</sheet>
+	</element>
+	<element name="Bite" type="Companion Action" source="Monstrous Compendium Volume Three: Minecraft Creatures" id="ID_WOTC_MCV3_COMPANION_ACTION_WOLF_OF_THE_OVERWORLD_BITE">
+		<description>
+			<p>Melee Weapon Attack: +4 to hit; reach 5 ft., one target. Hit: 7 (2d4 + 2) piercing damage. If the target is a creature, it must succeed on a DC 11 Strength saving throw or have the prone condition.</p>
```
Comment (NickVendel) [inline supplements/dnd-beyond/monstrous-compendium-3.xml:119]:
```suggestion
			<p><i>Melee Weapon Attack:</i> +4 to hit; reach 5 ft., one target. <i>Hit:</i> 7 (2d4 + 2) piercing damage. If the target is a creature, it must succeed on a DC 11 Strength saving throw or have the prone condition.</p>
```

File: supplements/dnd-beyond/legendary-magic-items.xml [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="dagger">Weapon</set>
+			<set name="rarity">Legendary</set>
+			<set name="weapon">Dagger</set>
+		</setters>
+	</element>
+
+	<element name="Tablet of Reawakening" type="Magic Item" source="Legendary Magic Items" id="ID_WOTC_LMI_MAGIC_ITEM_TABLET_OF_REAWAKENING">
+		<description>
+			<p>Witches of Rashemen created this Tiny stone tablet to counter the necromancy of the Red Wizards. As an action, you can cast <i>true resurrection</i> from the tablet. Once used in this way, the tablet turns to dust and is destroyed.</p>
```
[the remark this answers is not in the document]
Comment (FreshRoyal) [reply supplements/dnd-beyond/legendary-magic-items.xml:104]: I just copied and pasted the text from DDB, so yeah

**Pull request merged**